docs/content/development/extensions-contrib/redis-cache.md

```diff
@@ -8,9 +8,14 @@ Druid Redis Cache
 A cache implementation for Druid based on [Redis](https://github.com/antirez/redis).
 
 # Configuration
-Below are the configuration options known to this module:
+Below are the configuration options known to this module.
 
-|`runtime.properties`|Description|Default|Required|
+Note that just adding these properties does not enable the cache. You still need to add the `druid.<nodetype>.cache.useCache` and `druid.<nodetype>.cache.populateCache` properties for the nodes you want to enable the cache on.
```

Comment:
Something in here should ideally link to `../../configuration/caching.html`, where the properties are discussed.